micro_dl/train/losses.py

```diff
@@ -41,6 +42,12 @@ def kl_divergence_loss(y_true, y_pred):
     else:
         return K.sum(y_true * K.log(y_true / y_pred), axis=1)
 
+def dssim_loss(y_true, y_pred):
+    """Structural dissimilarity loss + L1 loss
+    0.8 * DSSIM + 0.2 * L1
+    """
+    mae = mean_absolute_error(y_true, y_pred)
+    return 0.8 * (1.0 - ssim(y_true, y_pred) / 2.0) + 0.2 * mae
```

Comment:
SSIM values are in range [-1, 1]. Wondering if you have to shift ssim to lie b/w 0 and 1:
1.0 - (1 + ssim(y_true, y_pred)) / 2.0